egs/librispeech/asr/simple_v1/prepare.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+import os
+from concurrent.futures import ProcessPoolExecutor
+from pathlib import Path
+
+import torch
+
+from lhotse import CutSet, Fbank, Mfcc, LilcomFilesWriter, WavAugmenter
+from lhotse.dataset import SpeechRecognitionDataset
+from lhotse.recipes.librispeech import download_and_untar, prepare_librispeech, dataset_parts_full
+
+print(dataset_parts_full)
+
+dataset_parts = ('dev-clean', 'test-clean', 'train-clean-100')
+
+corpus_dir = '/home/storage04/zhuangweiji/data/open-source-data/librispeech/LibriSpeech'
+output_dir = 'exp/data1'
+librispeech_manifests = prepare_librispeech(corpus_dir, dataset_parts,
+                                            output_dir)
+
+use_data_augmentation = False
+augmenter = WavAugmenter.create_predefined(
+    'pitch_reverb_tdrop',
+    sampling_rate=16000) if use_data_augmentation else None
+# It seems when spawning multiple Python subprocesses with the same sox handle it raises "std::runtime_error: Couldn't close file"
+# The issue seems to happen only in a Jupyter notebook on Mac OS X, hence the work around below.
+if use_data_augmentation:
+    num_jobs = 1
+else:
+    num_jobs = os.cpu_count()
+    torch.set_num_threads(1)
+    torch.set_num_interop_threads(1)
+
+num_jobs = 1
```

Comment:
Can this statement be deleted?

Reply:
I think yes, that was a workaround for launching wav augment with multiprocessing in a jupyter notebook on MacOS.

Reply:
Deleted as it was just a very basci scripts.